lldb/source/API/SBModule.cpp

```diff
@@ -671,3 +671,15 @@ void SBModule::GarbageCollectAllocatedModules() {
   const bool mandatory = false;
   ModuleList::RemoveOrphanSharedModules(mandatory);
 }
+
+const char *SBModule::GetName() const {
+  LLDB_INSTRUMENT_VA(this);
```

Comment:
There should be a newline after `LLDB_INSTRUMENT_VA`
```suggestion
  LLDB_INSTRUMENT_VA(this);

```